qiita_db/support_files/patches/24.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE qiita.reference ADD sortmerna_indexed_db_filepath bigint  ;
+
+CREATE INDEX idx_reference_2 ON qiita.reference ( sortmerna_indexed_db_filepath ) ;
```

Comment:
Is there a need for this index? I.e., will we ever be looking through the reference table looking for a particular filepath?